### PR TITLE
Addresses #173.

### PR DESCRIPTION
**Pull request by joseph-robertson**

### CODE REVIEW

**Comment by shorowit** [comment]
I would really like to have a formal fix from @kbenne for the OS PTAC issue rather than work around it.

File: measures/ResidentialAirflow/measure.rb [hunk 1]
```diff
@@ -1391,6 +1391,10 @@ def run(model, runner, user_arguments)
       
       if mech_vent.MechVentType == Constants.VentTypeBalanced
       
+        model.getZoneHVACEnergyRecoveryVentilators.each do |erv|
+          erv.remove
+        end
+
```
Comment (shorowit) [inline measures/ResidentialAirflow/measure.rb]: Seems like some unrelated changes here. Could you break this and the unit test changes into their own PR? 

Also, this doesn't strike me like the best place for this code; why not have this around [line 962](https://github.com/NREL/OpenStudio-BEopt/blob/afa9d45a7dffd6d8444dad5feda5a992f517d8b6/measures/ResidentialAirflow/measure.rb#L962)? And probably best to add checks based on the name of the object, as is commonly done elsewhere?

**Comment by shorowit** [comment]
Why is there not a single unit test that demonstrates the issue is resolved?

**Comment by joseph-robertson** [comment]
`test_retrofit_replace_furnace_room_air_conditioner` tests the scenario, but it passes because the error doesn't occur until EnergyPlus is run. I guess I could look at objects in the equipment list or something?

**Comment by shorowit** [comment]
Should now be fixed upstream.

**Pull request closed without merging**